subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/SwiftCompile.java

```diff
@@ -141,6 +142,7 @@ public SwiftCompile(
      */
     @InputFiles
     @SkipWhenEmpty
+    @IgnoreEmptyDirectories
     @PathSensitive(PathSensitivity.RELATIVE)
     public ConfigurableFileCollection getSource() {
```

Comment:
Why is this thing not an `AbstractNativeCompileTask` I wonder?

Reply:
You'll probably need to ask @big-guy, I don't know.

Reply:
The task `AbstractNativeCompileTask` is header-aware which makes no sense for Swift.